### DISCUSSION TITLE
Spike Sorting Data from Axion MEA system

### DISCUSSION THREAD

**Discussion by ajcheng1** [petersenpeter/CellExplorer#111] [General]
Is there a way to use CellExplorer on .spk, .raw, or .nex files? 
It seems like MEA data from axion biosystems (Maestro Pro) is closed off and I am unable to convert any of the files into .dat or .mat. Any suggestions?

**Comment by petersenpeter** [comment]
Hi 

Typically they provide export functions so I would expect there to be a way to extract the data. Are you sure that the .raw is not a binary format? Could you provide a link to the documentation of their file format (.spk, .raw .nex)?

**Reply by ajcheng1** [reply]
I attached a file that demonstrates how their spike sorting protocol works. 
[Spike Sorting Protocol - Axion BioSystems.pdf](https://github.com/user-attachments/files/16184062/Spike.Sorting.Protocol.-.Axion.BioSystems.pdf)
 
I believe the .raw file is binary, but it is unsorted so I might have to go through kilosort in order to plug in the data into cellexplorer.

**Reply by ajcheng1** [reply]
Hi Peter,

I can export the function to a .plx, .pl2, .nex or .nex5. The issue is that I need to be able to extract data from each individual waveforms from cell explorer. How should I go about it? 

Some of Plexon Offline Sorter software allows for matlab conversion, but it just reads .plx file for me to copy and paste into a .mat file, not necessarily the individual waveforms
